GameData/RealismOverhaul/Engine_Configs/AJ10_Adv_Config.cfg

```diff
@@ -301,40 +301,51 @@
 		}
 	}
 }
-// 47 flights, 5 failures. 
+
+//Titan IIIA: 2 flights, 1 failures. (1 unrelated)
+//Titan IIIC: 12 flights, 2 failures. (1 cycle, 1 restart) (1 engine failed to shut down. Uneven thrust destroyed transtage)
+//Titan III(23)C: 21 flights, 1 failure. (1 unrelated)
+//Titan 34D/Transtage: 7 flights, 1 failure. (1 restart)
+
 // Assuming 3 ignitions average since transtage was used exclusively for GSO/GEO
-// 131 ignitions, 0 ignition failures.
+// 2 AJ10s per transtage
+// 84 ignitions, 0 failures
+// 80 cycles, 1 failures
+// 156 restarts, 2 failures
 @PART[*]:HAS[@MODULE[ModuleEngineConfigs]:HAS[@CONFIG[AJ10-138]],!MODULE[TestFlightInterop]]:BEFORE[zTestFlight]
 {
 	TESTFLIGHT
 	{
 		name = AJ10-138
 		ratedBurnTime = 450			//no extra burn time, AJ10 switched to ablative chambers
-		ignitionReliabilityStart = 0.992424
-		ignitionReliabilityEnd = 0.998485
-		cycleReliabilityStart = 0.893617
-		cycleReliabilityEnd = 0.978723
+		ignitionReliabilityStart = 0.988589
+		ignitionReliabilityEnd = 0.997718
+		cycleReliabilityStart = 0.978395
+		cycleReliabilityEnd = 0.995679
 		techTransfer = AJ10-104,AJ10-118E:15	// New dual design lead to many early growing pains issues
 	}
 	@MODULE[ModuleEngineConfigs] { @CONFIG[AJ10-138] { %ratedBurnTime = #$/TESTFLIGHT[AJ10-138]/ratedBurnTime$ } }
 }
-//Delta 300: 3 flights, 1 failure
+
+//Delta 300: 3 flights, 1 failure (1 unrelated)
 //Delta 900: 2 flights, 0 failures
 //Delta 1604: 2 flights, 0 failures
 //Delta 1900: 1 flight, 0 failures
-// 8 flights, 1 failure
-// mostly used for sun-synch orbits, assuming 1.5 ignitions average
-//12 ignitions, 0 failures
+
+//mostly used for sun-synch orbits, assuming 2 ignitions average
+//8 ignitions, 0 failures
+//7 cycles, 0 failures
+//7 restarts, 0 failures
 @PART[*]:HAS[@MODULE[ModuleEngineConfigs]:HAS[@CONFIG[AJ10-118F]],!MODULE[TestFlightInterop]]:BEFORE[zTestFlight]
 {
 	TESTFLIGHT
 	{
 		name = AJ10-118F
 		ratedBurnTime = 450
-		ignitionReliabilityStart = 0.923077
-		ignitionReliabilityEnd = 0.984615
-		cycleReliabilityStart = 0.875000	// 1 failure out of 8 flights
-		cycleReliabilityEnd = 0.975000
+		ignitionReliabilityStart = 0.953125
+		ignitionReliabilityEnd = 0.990625
+		cycleReliabilityStart = 0.906250
+		cycleReliabilityEnd = 0.981250
```

Comment:
I am very doubtful that the ending reliability for 118F should be *lower* than for the -138. I really think we have to use judgment here, not just run the numbers, because otherwise we unfairly penalize rarely-flown variants.

Reply:
Oops, I tried to use other data for engines with lower sample sizes. I must have forgotten this one

Reply:
:+1: